keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/UnhealthyNotificationListener.kt

```diff
@@ -48,7 +48,7 @@ class UnhealthyNotificationListener(
         val unhealthyDuration = unhealthyRepository.durationUnhealthy(event.resource)
         if (unhealthyDuration > config.minUnhealthyDuration) {
           publisher.publishEvent(
-            NotificationEvent(RESOURCE, event.resource.id, UNHEALTHY_RESOURCE, message(event, unhealthyDuration))
+            UnhealthyNotification(RESOURCE, event.resource.id, UNHEALTHY_RESOURCE, message(event, unhealthyDuration))
```

Comment:
Passing the `UNHEALTHY_RESOURCE` type is redundant with the constructor here, right?

Reply:
yes, removed it!